libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -133,3 +152,45 @@ class AP_MotorsHeli_RSC {
     // calculate_desired_throttle - uses throttle curve and collective input to determine throttle setting
     float           calculate_desired_throttle(float collective_in);
 };
+
+class RSCThrCrvInt16Param {
+public:
+    RSCThrCrvInt16Param(void);
+
+    static const struct AP_Param::GroupInfo var_info[];
+
+    void set_thrcrv_enable(int8_t setenable) {enable = setenable; }
+    float * get_thrcrv() {
+        static float throttlecurve[5];
+        for (uint8_t i = 0; i < 5; i++) {
+            throttlecurve[i] = (float)thrcrv[i] * 0.001f;
+        }
+        return throttlecurve;
+    }
+
+private:
+    AP_Int8   enable;
+    AP_Int16  thrcrv[5]; // throttle value sent to throttle servo at 0, 25, 50, 75 and 100 percent collective
+
+};
+
+class RSCGovFloatParam {
+public:
+    RSCGovFloatParam(void);
+
+    static const struct AP_Param::GroupInfo var_info[];
+
+    void set_gov_enable(int8_t setenable) {enable = setenable; }
+    int16_t get_setpoint() { return setpoint; }
+    float get_disengage() { return disengage; }
+    float get_droop_setting() { return droop_setting; }
+    float get_tc() { return tc; }
+
+private:
+    AP_Int8   enable;
+    AP_Float  disengage;    // sets the throttle percent where the governor disengages for return to flight idle
```

Comment:
I mentioned earlier as well but perhaps we could make the parameter variables names a bit more specific to help future developers?